packages/cli/src/dev.ts

```diff
@@ -0,0 +1,143 @@
+import Ajv, { Schema } from 'ajv';
+import addFormats from 'ajv-formats';
+import chokidar from 'chokidar';
+import * as fs from 'fs';
+import * as yaml from 'js-yaml';
+import { Miniflare } from 'miniflare';
+import ora from 'ora';
+import * as path from 'path';
+
+import { buildScriptFromManifest } from './build';
+import { prettyPath } from './files';
+import { getDefaultManifestPath, resolveFile, resolveIntegrationManifestPath } from './manifest';
+import { getAPIClient } from './remote';
+import { createDevTunnel } from './tunnel';
+
+export const GITBOOK_DEV_SERVER_PORT = 8787;
```

Comment:
This port might be taken, there is a node module to find an empty port. I think it'll be better than having a user option.

Reply:
Agree with the suggestion 👍